src/components/pages/ClinHome.vue

```diff
@@ -1936,6 +1939,11 @@ export default {
 
       }
 
+      if(!customData.hasOwnProperty("bedFileUrl")){
+        bool = false;
+        message = "Could not interpret bed file URL. (\"bedFileUrl\": \"\")";
+
+      }
```

Comment:
Hi @WilliamRichards2017 , I have added a new field for adding a bed file. Based on your previous checks for validation, I had added the similar check for the bed URL field. Please review it when you get a chance. 
You can use this URL to test for custom bed file: https://raw.githubusercontent.com/iobio/clin.iobio/custom-data-bam/src/data/20130108.exome.targets.bed

You can refer to this thread for more info: https://github.com/iobio/clin.iobio/issues/228